engine/platform/src/platform_window_glfw3.cpp

```diff
@@ -181,6 +190,12 @@ namespace dmPlatform
 
     static PlatformResult OpenWindowOpenGL(dmWindow* wnd, const WindowParams& params)
     {
+    #if defined(__linux__) && defined(__aarch64__) && !defined(ANDROID)
+        glfwWindowHint(GLFW_CLIENT_API, GLFW_OPENGL_ES_API);
+        glfwWindowHint(GLFW_CONTEXT_CREATION_API, GLFW_EGL_CONTEXT_API);
+        glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 2);
```

Comment:
Not knowing what the different targets support, but I would think it's not unreasonable to think one of them might support opengles 3?

E.g. the android code does this to detect it:
```
        const char* version = (char *) glGetString(GL_VERSION);
        if (strstr(version, "OpenGL ES 2.") != 0) {
            context->m_IsGles3Version = 0;
        } else {
            context->m_IsGles3Version = 1;
        }
```

Also, I'd expect the gles variables to be set accordingly:
```
context->m_IsGles3Version = ...;
context->m_IsShaderLanguageGles = ...;
```